NineChronicles.DataProvider/Configuration.cs

```diff
@@ -86,5 +86,7 @@ public class Configuration
 #pragma warning restore SA1011 // Closing square brackets should be spaced correctly
 #nullable disable
         public string MySqlConnectionString { get; set; }
+
+        public bool Preload { get; set; }
```

Comment:
Do we need this option instead of forcing skipping preload?

Reply:
Probably not, I'll change this option to false by default. :pray: